src/commands/shutdown.json

```diff
@@ -1,6 +1,7 @@
 {
     "SHUTDOWN": {
         "summary": "Synchronously save the dataset to disk and then shut down the server",
+        "complexity": "O(N) where N is the total number of keys in all databases",
```

Comment:
Or maybe

```suggestion
        "complexity": "O(1) when not saving the data, otherwise O(N) where N is the total number of keys in all databases",
```

Reply:
I think I'll prefer something like "O(N) when saving, O(1) otherwise".

Reply:
I think we should mention what's "N" when we specify O(N).

Reply:
yes, I said "something like", not exactly this. I'll add a commit tomorrow.